### ISSUE TITLE
Netrunner Blackbird: codename changed and breaks updating

### ISSUE THREAD

**Issue by star-buck** [blue-systems/netrunner-issues-old#194]
happened this morning:
![Screenshot_20190708_084657](https://user-images.githubusercontent.com/840070/60788634-72b4e900-a15d-11e9-963e-36b70bffafb1.png)

**Comment by ScarlettGatelyMoore** [comment]
yeah I had fun with our docker images this weekend due to this very problem. The only thing I found to work was to apt update and accept the prompt in the console. There was no automated way to fix it as apt -y update didn't even work.

**Comment by star-buck** [comment]
Why is this happening exactly?

**Comment by ScarlettGatelyMoore** [comment]
Apparently it is the after affects of them renaming *-updates to *-security aka bullseye-security. https://www.mail-archive.com/debian-security@lists.debian.org/msg41224.html

This has not been fun for me.

**Comment by ScarlettGatelyMoore** [comment]
But with that said, as far as our ci we are now golden and back up and running.
